src/popup/components/ErrorKeyRevoked.tsx

```diff
@@ -164,16 +164,15 @@ const ReconnectScreen = ({
         requestReconnect();
       }}
     >
-      <div className="space-y-1 text-sm">
+      <div className="space-y-2 text-sm">
         <p className="px-2">
           Reconnecting to wallet:{' '}
           <span className="underline">{info.walletAddress?.id}</span>
         </p>
         <p className="px-2">
-          <strong>Before</strong> you reconnect, copy the public key below and
-          add it to your wallet.
+          <strong>Reconnect</strong> to automatically add the key to your
+          wallet.
```

Comment:
We should discuss this in the call: giving the users option to choose how to add the key - automatically or manually.

Reply:
Yes, we'll do that, reusing same consent screen. But keeping PR smaller atm.

Reply:
But still good to discuss in call also.

Reply:
In either case, we should not be showing this screen.
The auto-key-add-reconnect should have been triggered from `MainScreen` directly; or lead to consent screen followed by previous manual-key add screen.